src/script/entity/message/Message.js

```diff
@@ -18,6 +18,7 @@
  */
 
 import ko from 'knockout';
+import moment from 'moment';
```

Comment:
why?

Reply:
It was published on the global scope before (the old logger needed it). 
Now every class needs to import it manually